concepticondata/concepticon.tsv

```diff
@@ -3801,3 +3801,6 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3815	ALMOND	Agriculture and vegetation	The edible and cultivated fruit of the tree Prunus dulcis in the family Rosaceae. It resembles a nut and can be sold shelled or unshelled.	Person/Thing	
 3816	WAXING MOON	The physical world	The phase in the lunar cycle between new moon and full moon when the illuminated area increases.	Person/Thing	
 3817	WANING MOON	The physical world	The phase in the lunar cycle between full moon and new moon when the illuminated area decreases.	Person/Thing	
+3818	ASTRINGENT	Food and drink	A property describing a substance which causes the skin, membranes, or other tissue to contract. Tannins in unripe fruit are known to function as astringents.	Property	
+3819	HANDSPAN	The body	The distance between splayed out thumb and little finger or index finger of a human hand. Formerly used as a unit of measurement or for roughly gauging small lengths around 20 centimeters.	Person/Thing	
+3820	BROAD BEAN	Agriculture and vegetation	Bean of the plant Vicia faba, also known as fava bean. It is typically large, of a pale green colour, and edible.		
```

Comment:
The last column is missing "Person/Thing" in this line.

Reply:
You're right, thanks!